Add Clavister NetWall deployment How-To (#308)
citynetwork/docs: docs/howto/marketplace/clavister/deploy.md

---
description: How to deploy a Clavister NetWall instance in Cleura Cloud
---

# Creating a firewall instance

> Assuming you already have an [account in {{brand}}](../../getting-started/create-account.md), you may log in to the [{{gui}}](https://cleura.cloud), visit the Marketplace, and deploy any of the available applications or services.

Reviewer: ```suggestion
Assuming you already have an [account in {{brand}}](../../getting-started/create-account.md), you may log in to the [{{gui}}](https://{{gui_domain}}), visit the Marketplace, and deploy any of the available applications or services.
```

> In this guide, we demonstrate the deployment of a Clavister NetWall application firewall.

Reviewer: ```suggestion
This guide covers the deployment of a Clavister NetWall application firewall.
```

## Step-by-step deployment

In the left-hand side vertical pane of the {{gui}}, expand the *Marketplace* category and click on *Discover Apps and Services*.
In the central pane, you will see all available applications and services.
Locate the *Clavister NetWall* box and click the green *View* button.

![Select the Clavister NetWall application](assets/new-firewall/netwall-01.png)

You will see the *Clavister NetWall* information page, where you can learn more about its features and obtain pricing information.
Click the orange *Deploy this App* button to start the deployment process.

![Start the firewall deployment process](assets/new-firewall/netwall-02.png)

> The firewall is hosted on a Nova VM, so now you may select a region, a name, a flavor, a public network, and a keypair for it.

Reviewer: Maybe add a cross-reference to the "Creating new servers" Nova how-to here?

In the central pane, watch the firewall VM row.
The animated icon at the left marks the deployment progress.

![Check the deployment progress](assets/new-firewall/netwall-04.png)

When the deployment is complete, you will see a white check mark in a green circle at the left of the firewall VM row.

![The firewall is deployed](assets/new-firewall/netwall-05.png)

## Logging into the NetWall dashboard

You need to know the firewall VM's public IP address and the automatically generated password for the `admin` user.
For that, make sure you are in the *Provisioned Apps* pane.
Click on the firewall row to expand it, and select the *Stack Output* tab.

![Locate the public IP and the admin user password](assets/new-firewall/netwall-dashboard-01.png)

In the *gateway_password* row, click the icon in the *Action* column.
A pop-up window appears, and you may click the blue *Copy Output!* button to copy the password to the clipboard.
Close the pop-up window by clicking on the *Back* button.

![Copy the admin user password to the clipboard](assets/new-firewall/netwall-dashboard-02.png)

> Similarly, click on the *gateway_public_ip* to get a pop-up window with the public IP.

Reviewer: I *think* we're missing a word here:

```suggestion
Similarly, click on the *gateway_public_ip* row to get a pop-up window with the public IP.
```
